README.md

```diff
@@ -127,6 +128,15 @@ January 1, 2019, 00:00:00 UTC and will stop firing January 31, 2019, 23:59:00 UT
 
  **Note**: The parameter `maxTriggers` is deprecated and will be removed soon.  To stop the Trigger, use the `stopDate` parameter.
 
+- `strict` (*optional*): A boolean value that decides to add a few seconds to the Trigger. This work with only five-field cron alarms.
+
+  - If it's true, the Trigger will fire at the top of the hour/minute (\**:**:00).
+  - Otherwise, the Trigger will fire after the specific seconds (\**:**:00-59).
+
+The delay is determined by the hash value of the Trigger's name, so it keeps the same interval before and after the (re)deployment.
+
+**Note** This option can be helpful to avoid thundering herds when the second-unit errors are not critical.
```

Comment:
```suggestion
**Note** This option can be helpful to avoid thundering herds when the same firing interval is more important than the seconds-unit precision.
```